Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node
+ARG SERVICE_NAME
+
+# These steps ensure that npm install is only run when package.json changes
+RUN mkdir -p /usr/src/$SERVICE_NAME
+ADD ./package.json /usr/src/$SERVICE_NAME/package.json
+WORKDIR /usr/src/$SERVICE_NAME
+RUN npm install --production
+
+ADD . /usr/src/$SERVICE_NAME
```

Comment:
I guess the Dockerfile isn't really needed in the first place. If you're going to be using this as part of an application, you are going to run `exo-setup` at some point before running the app. The Dockerfile will be brought in then, so it doesn't need to be included as part of this repo itself, do you agree?